### ISSUE TITLE
switch runs with key bindings

### ISSUE THREAD

**Issue by lunebellec** [SIMEXP/dashQC_fmri#54]
In the motion tab it would be useful to be able to switch runs using a key binding

**Comment by bpinsard** [comment]
I agree. Also when switching with the dropdown menu, I am sometime not sure when it has changed (as it is heavy files to load). Maybe a discreet load indicator in a corner of the screen would be useful.
